src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { useAlertHandlerContext } from "../../../contexts/alert_handler";
+
+const AlertManager = () => {
+  const AlertHandlerContext = useAlertHandlerContext();
+
+  if (AlertHandlerContext.visible === true) {
+    setTimeout(AlertHandlerContext.closeAlert, 5000);
+  }
+
+  //AlertHandlerContextから直接値が取り出せる
+  console.log(AlertHandlerContext.visible);
+  AlertHandlerContext.setAlert("message"); //Alertの表示

```

Comment:
ここが原因でさまざまなエラーが起こっています。
1. Cannot update a component (`x`) while rendering a different component.
2. returnがない

１の解決法としてuseEffectを使うというものがあります。
参考：https://chaika.hatenablog.com/entry/2021/06/24/083000
２の解決法はおそらく忘れていると思うのですが　Alert　を呼び出せば解決するはずです。

```suggestion
  const AlertHandlerContext = useAlertHandlerContext();

  useEffect(() => {
    if (AlertHandlerContext.visible === true) {
      setTimeout(() => {
        AlertHandlerContext.closeAlert();
      }, 5000);
    }
  }, [AlertHandlerContext]);

  // //AlertHandlerContextから直接値が取り出せる
  console.log(AlertHandlerContext.visible);
  return (
    <Alert
      SText={AlertHandlerContext.message}
      visible={AlertHandlerContext.visible}
    />
  );

```

Reply:
以上はとりあえずエラーを解消しただけなのでここからさらに仕様に近づけるようにしてください